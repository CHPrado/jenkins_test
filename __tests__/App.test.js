import React from "react";
import { mount } from 'enzyme';
import App from "../src/App";

describe('Testing App Component', () => {
  it('should render correctly', () => {
    const wrapper = mount(
     <App />
    );

    expect(wrapper.debug()).toMatchSnapshot();
  });
});