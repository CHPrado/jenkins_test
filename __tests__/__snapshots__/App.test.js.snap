// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component should render correctly 1`] = `
"<App>
  <div className=\\"App\\">
    <h1>
      Hello World
    </h1>
  </div>
</App>"
`;
